source/ed/data.cpp

```diff
@@ -600,45 +600,48 @@ void Data::pick_up_drop_of_on_borders() {
         auto* first_st = vj->stop_time_list.front();
         auto* last_st = vj->stop_time_list.back();
 
-        first_st->drop_off_allowed = false;
-        last_st->pick_up_allowed = false;
-
         /*
          * In the example below we are assuming that VJ:1 and VJ:2 are sharing the same block_id.
          *
          * The stay-in section is allowed with 2 configurations:
          *  - when two VJ share the same stop point with similar stop times (example 1)
          *  - when two VJ are joined on 2 different stop points with consecutive stop times (example 2)
```

Comment:
```suggestion
         *  - when two VJ are joined on 2 different stop points with consecutive stop times  (example 1)
         *  - when two VJ share the same stop point with similar stop times (example 2)
```
I think you exchanged the examples at some point

Reply:
well spotted :+1: